setup.py

```diff
@@ -4,7 +4,7 @@
 
 setup(
     name = "OpenFisca-Country-Template",
-    version = "3.12.9",
+    version = "3.12.10",
```

Comment:
In the context of the Template, I believe that switching from CircleCI to GitHub Actions should be a minor, not a patch 🙂

Reply:
Okay :) so `3.13.0` ?